src/ZODB/FileStorage/fspack.py

```diff
@@ -445,11 +442,10 @@ def close_files_remove():
             # pack didn't free any data.  there's no point in continuing.
             close_files_remove()
             return None
-        self._commit_lock_acquire()
+        self._commit_lock.acquire()
```

Comment:
Since acquire/release all occur in the same method, wouldn't this be clearer using `with self._commit_lock:`?